tests/ml/sklearn_test.py

```diff
@@ -213,3 +214,25 @@ def test_sklearn_incremental_predictor_serialize(tmpdir):
 
     pred_in_memory = incremental.predict(df_test)
     np.testing.assert_array_almost_equal(pred_in_memory, df_test.pred.values, decimal=1)
+
+
+@pytest.mark.parametrize("batch_size", [6789, 10_000])
+@pytest.mark.parametrize("num_epochs", [1, 5])
+def test_sklearn_incremental_predictor_partial_fit_calls(batch_size, num_epochs):
+    df = vaex.example()
+    df_train, df_test = df.ml.train_test_split(test_size=0.1, verbose=False)
+
+    features = df_train.column_names[:6]
+    target = 'FeH'
+
+    N_total = len(df_train)
+    num_batches = (N_total + batch_size - 1) // batch_size
+
+    incremental = IncrementalPredictor(model=mock.Mock(),
+                                       features=features,
+                                       batch_size=batch_size,
+                                       num_epochs=num_epochs,
+                                       shuffle=False,
+                                       prediction_name='pred')
+    incremental.fit(df=df_train, target=target)
+    assert incremental.model.partial_fit.call_count == num_batches * num_epochs
```

Comment:
This is a pretty trivial assertion. I am more worried about the # of rows being passed being equal to len(df_train).